caffeinated/admin/extend/registrations/Extend_Registrations_Admin_Page.core.php

```diff
@@ -796,6 +796,14 @@ protected function _registration_checkin_list_table()
         $reg_id = isset($this->_req_data['_REG_ID']) ? $this->_req_data['_REG_ID'] : null;
         /** @var EE_Registration $registration */
         $registration = EEM_Registration::instance()->get_one_by_ID($reg_id);
```

Comment:
plz sanitize with `absint()` if `$reg_id` is not sanitized by the models

Reply:
that's actually not part of this PR, it turns out... and yes the models do sanitize data also BUT I've heard it's best to sanitize ASAP and layers of security is best anyway.